src/analysis/processing/qgsalgorithmcellstatistics.cpp

```diff
@@ -398,11 +398,11 @@ QString QgsCellStatisticsPercentileAlgorithm::shortHelpString() const
                       "   <li>Exclusive linear interpolation (PERCENTILE.EXC)</li>"
                       "</ul> "
                       "While the output value can stay the same for the nearest rank method (obtains the value that is nearest to the "
-                      "specified percentile), the linear interpolation method return unique values for different percentiles. Both interpolation "
+                      "specified percentile), the linear interpolation methods return unique values for different percentiles. Both interpolation "
                       "methods follow their counterpart methods implemented by LibreOffice or Microsoft Excel. \n\n"
```

Comment:
same here. Any refs to those implementation?

Reply:
What do you mean by references? References to the specific implementations?